xml/decoders/zimo/Choice-F0F28.xml

```diff
@@ -106,4 +108,96 @@
       <enumChoice choice="F28">
         <choice>F28</choice>
       </enumChoice>
+      
+      
+      <!--  Inverted Function Keys  add 128 to standard value  -->
+      
+      <enumChoice choice="Invert F0" value="157">
+        <choice>Invert F0</choice>
+      </enumChoice>
+      <enumChoice choice="Invert F1" value="129">
+        <choice>Invert F1</choice>
+      </enumChoice>
+      <enumChoice choice="Invert F2">
+        <choice>Invert F2</choice>
+      </enumChoice>
```

Comment:
No need to change, but a question. F0 and F1 has value 157 and 129. But F2 - F28 doesn't have a value. Does that matter? I don't know what the value is about so I have no idea.

Reply:
Doesn't matter -  JMRI assumes they follow sequentially the value set immediately before.   
So, "Invert F1" = 129,  next is "Invert F2" = 130, etc...  

Earlier version of file uses same approach for values from F0 (=29) then F1=1, and the remainder F2 to F28 were not declared as value.   

I also checked it was working on my local JMRI copy, with correct values appearing in the CV list pane.